crates/transform-gizmo-bevy/src/lib.rs

```diff
@@ -88,6 +87,10 @@ pub struct GizmoOptions {
     /// using a single gizmo. If `false`, each target
     /// has its own gizmo.
     pub group_targets: bool,
+    /// Allows you to provide a custom viewport rect, which will be used to
+    /// scale the cursor position. By default, this is set to `None` which means
+    /// the full window size is used as the viewport.
```

Comment:
Thanks, looks great! I might change the wording here a bit, since one might assume that simply by setting the custom viewport rect, they can render the gizmo to the specified rect (even though it does say "will be used to scale the cursor position"). Technically the full window size is still used as the viewport, but only the interaction is affected.